prime-router/build.gradle.kts

```
@@ -299,14 +303,21 @@ tasks.register<ResolveTask>("generateOpenApi") {
     classpath = sourceSets["main"].runtimeClasspath
     buildClasspath = classpath
     resourcePackages = setOf("gov.cdc.prime.router.azure")
-    outputDir = file("docs/api/generated")
+    outputDir = apiDocsSpecDir
+    dependsOn("compileKotlin")
 }
 
-/**
- * Generate OpenAPI spec right after build
- */
-tasks.named("build") {
-    finalizedBy("generateOpenApi")
+tasks.register<Copy>("copyApiSwaggerUI") {
+    // copy generated OpenAPI spec files (api.yaml for now)
```

```suggestion
    // copy generated OpenAPI spec files
```

I don't think we need this part of the comment here since it should be out of date shortly?